cypress/support/generate.js

```diff
@@ -0,0 +1,21 @@
+const { faker } = require('@faker-js/faker');
+
+export const generateUser = () => {
+  const name = faker.internet.userName()
+    .replace(/[^a-zA-Z0-9]/g, '').substring(0, 40);
+
+  return {
+    username: name,
+    email: `${name}@gmail.com`,
+    password: 'Test1234'
+  };
+};
+
+export const generateeArticle = () => {
```

Comment:
There is a typo in the function name 'generateeArticle'. It should be 'generateArticle'.